src/client/cli/cmd/stop.cpp

```diff
@@ -84,7 +84,8 @@ mp::ParseCode cmd::Stop::parse_args(mp::ArgParser* parser)
     QCommandLineOption time_option({"t", "time"}, "Time from now, in minutes, to delay shutdown of the instance",
                                    "time", "0");
     QCommandLineOption cancel_option({"c", "cancel"}, "Cancel a pending delayed shutdown");
-    parser->addOptions({all_option, time_option, cancel_option});
+    QCommandLineOption force_option("force", "Force switching the instance off");
```

Comment:
I would prefer if we added a little more explanation to let users know that this equates to pulling the plug on the machine and that may result in corruption of the VM.

Reply:
Agreed, will modify the wording here.  Thanks!

Reply:
Ok, I modified this.  Please let me know what you think.  Thanks!

Reply:
Yep, much better. Thank you!